Omnibus.md

```diff
@@ -4006,9 +4010,9 @@ above and beyond a normal player.
   date of the event. Also within that month, you must also send a summary of
   reports of rules violations and the conclusion of investigations into them, as
   well as all consequences levied, to the [Arbitration
-  Committee](#arbitration-committee) if any reports were filed. EHs and the AC
-  have the responsibility to maintain the privacy of those involved in reports
-  to the extent possible.
+  Committee](#arbitration-committee) if any reports were filed. EHs and the
+  [Abitration Committee]{.new} have the responsibility to maintain the privacy
+  of those involved in reports to the extent possible.
```

Comment:
"to the extent possible"

Not something we can fix, but I really hate how ambiguous this is.  Proposal for next year?